src/shared/components/query/CaseSetSelector.tsx

```diff
@@ -68,7 +76,7 @@ export default class CaseSetSelector extends QueryStoreComponent<{}, {}>
 				<ReactSelect
 					value={this.store.selectedSampleListId}
 					options={this.caseSetOptions}
-					clearable={this.store.selectedSampleListId != this.store.defaultSelectedSampleListId}
+					filterOption={(opt: ReactSelectOptionWithName, filter)=>_.includes(opt.textLabel.toLowerCase(), filter)}
```

Comment:
I actually do think, strictly speaking, a unit test would be useful here - pull the lambda from filterOption into a function that can be unit tested. That's good React practice anyway, since with a lambda as a prop it will rerender every time even if the props didn't really change.